<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html xmlns="http://www.w3.org/1999/xhtml"><!-- InstanceBegin template="/Templates/homer.dwt" codeOutsideHTMLIsLocked="false" -->
<head>
    <meta name="keywords" content="organ-specific" />
    <meta name="description" content="Homer£º Human Organ-specific Molecular Electrical Repository" />
    <meta name="author" content="Jake Y. Chen, Fan Zhang" />
    <!-- InstanceBeginEditable name="doctitle" -->
    <title>Homer: Human Organ-specific Molecular Electrical Repository</title>
    <!-- InstanceEndEditable -->
    
    <!-- InstanceBeginEditable name="head" -->
    <style type="text/css" media="screen">
        @import "site_jui.css";
    </style>
	<style type="text/css" media="screen">
        @import "demo_table_jui.css";
    </style>
    <style type="text/css" media="screen">
        @import "jquery-ui-1.7.2.custom.css";
    </style>
    <style type="text/css" media="screen">
        /*
			 * Override styles needed due to the mix of three different CSS sources! For proper examples
			 * please see the themes example in the 'Examples' section of this site
			 */
        .dataTables_info
        {
            padding-top: 0;
        }
        .dataTables_paginate
        {
            padding-top: 0;
        }
        .css_right
        {
            float: right;
        }
        #example_wrapper .fg-toolbar
        {
            font-size: 0.8em;
        }
        #theme_links span
        {
            float: left;
            padding: 2px 10px;
        }
    </style>
    <!-- InstanceEndEditable -->
</head>
<body id="index" class="grid_2_3">
    <div id="fw_container">
        <div id="fw_header">
            <h1>
                <a href="index.html">
                    <img src="homer.jpg" alt="Homer logo" width="77px" height="75px">Homer</a>
          </h1>
            <!-- InstanceBeginEditable name="headeditregion" -->
			<div style="border-bottom-style:solid; border-bottom-width:3px; border-bottom-color:#c3d1ec">
            <ul>
                <li><a href="index.html">Home</a></li>
                <li><a href="browse.html">Browse</a></li>
                <li><a href="advsearch.html">Advanced Search</a></li>
                <li><a href="download.html">Download</a></li>
                <li><a class="active" href="help.html">Help</a></li>
                <li><a href="statistics.html">Statistics</a></li>
                <li><a href="about.html">About Us</a></li>
            </ul>
                    
            <div class="css_clear"></div>
            </div>

            <ul class="submenu submenu_styling">
            	<li><a href="#">Help</a></li>
			</ul>                 

            <div class="css_clear"></div>
            
            <!-- InstanceEndEditable -->
        </div>
        <div id="fw_content">
          <!-- InstanceBeginEditable name="EditRegion1" -->
          <div style="width: 100%; border: 1px solid #c3d1ec; -moz-border-radius: 1em;">
            <p>&nbsp;</p>
                <table width="100%" height="57">
                    <tr>
                        <td width="72">&nbsp;                            
                        </td>
                        <td width="744">
                       
                       <h3> Frequently Asked Questions </h3>    
	<p><a href="#para1">What is Organ-specific Marker?</a></p>
	<p><a href="#para2">What is the Homer?</a></p>
	<p><a href="#para3">How to use it?</a></p>
<p><a name="para1"></a>The organ-specific  marker is defined as an organ-specific gene/protein that is also implicated in human diseases related to the organ. It is an indicator to measure the wellness or normal  functioning of a tissue in a respective organ. It introduces effective  intervention techniques towards preventive and predictive medicine. It can  improve sensitivity and target organ specificity for most peripheral clinical  pathology.</p>
    <p><a name="para2"></a>The Human Organ-specific Molecule Electronic Repository (Homer) is a comprehensive database covering about 22809 proteins, 52 organs, and 4290 diseases integrated and filtered from organ-specific protein/genes and disease databases like dbEST[1], TiSGeD[2], HPA[3],  CTD[5], and Disease Ontology[6]. It is the first comprehensive database that can be used to analyze, identify, and characterize organ-specific biomarkers in association with disease-organ and disease-protein relationship. The database has a web user interface that allows users to query organ-specific genes/proteins by gene, protein, organ or disease, to explore the histogram of an organ-specific gene/protein, and to identify disease-related organ-specific genes by browsing the disease online. </p>
    <p><a name="para3"></a>The user interfaces of the web-based online version of Homer supports multiple search options. The first search option shown in the Figure 1 allows users to query organ-specific gene/protein by a gene or protein or multiple genes or proteins. It can also browse Gene Symbol, Organ Specificity, Source, Significance(pvalue and zscore), and Disease Relevance for each gene/protein(Figure 2). Users can further explore the histogram of organ-specific gene/protein across the 52 organs by clicking the histogram icon in the organ specificity column (Figure 3). In the last column, users will be redirected to the disease relevance webpage when clicking on the disease relevance icon (Figure 4). Secondly, users can query organ-specific gene/protein by Advanced Search(Figure 5). Lastly,  users can query organ-specific gene/protein by browsing human organ system(Figure 6) or disease ontology (Figure 7). The organ-specific biomarkers are freely available for download in tab-delimited format on the <a href="download.html">download</a> page or individual inqury page.</p>
    <p><img src="images/index.png" alt="searchbygeneprotein" width="100%" height="100%" /></p>
    <p>Figure 1: query organ-specific gene/protein by a gene or protein or multiple genes or proteins. This is a search to determine the  organ-specific genes/proteins. Users can use different identifiers to  query the database. The recommended terms are Unigene  ID¡¯s, Entrenz gene  ID¡¯s, Gene name, Uniprot  ID¡¯s, and IPI ID¡¯s. It's CASE SENSITIVE.</p>
<p><img src="images/searchresult.png" alt="pvalueandzscore" width="100%" height="100%" /></p>
<p>Figure 2: The result of querying organ-specific gene/protein by gene or protein. 
      The query ¡°APOB¡± directed to the respective protein whose corresponding Genes (Unigene ID), Organ Specificity, P-value, Zscore, disease relevance, and the database source is presented. 
  A click on the histogram icon provides a histogram graphing the expression of the queried gene in all the available organs (Figure 3).      Clicking on the disease relevance icon users will be directed to the disease relevance webpage where the related diseases and associated organs could be mapped (Figure 4).</p>
<p><img src="images/histogram.png" alt="figure3" width="100%" height="100%" /></p>
    <p>Figure 3: histogram of the expression of the queried gene or protein</p>
    <p><img src="images/disease relevance.png" alt="figure3" width="100%" height="100%" /></p>
    <p>Figure 4: disease relevance of the queried gene or protein</p>
    <p><img src="images/advancedsearch.png" alt="figure3" width="100%" height="100%" /></p>
    <p>Figure 5: advanced search </p>
    <p></p>
<p><img src="images/browse.png" alt="querybyorgan" width="100%" height="100%" /></p>
    <p>Figure 6: browsing by human organ system</p>
    <p><img src="images/browse2.png" alt="querybydisease" width="100%" height="100%" /></p>
    <p>Figure 7: browsing by disease ontology</p>

<p>&nbsp;</p>
<h3>References:</h3>
<p>1.	Boguski MS, Tolstoshev CM, Bassett DE, Jr.: Gene discovery in dbEST. Science 1994, 265(5181):1993-1994.<br />
      2.	Xiao SJ, Zhang C, Zou Q, Ji ZL: TiSGeD: a database for tissue-specific genes. Bioinformatics 2010, 26(9):1273-1275.<br />
      3.	Uhlen M, Oksvold P, Fagerberg L, Lundberg E, Jonasson K, Forsberg M, Zwahlen M, Kampf C, Wester K, Hober S et al: Towards a knowledge-based Human Protein Atlas. Nat Biotechnol 2010, 28(12):1248-1250.<br />
      4.	Saha S, Harrison SH, Shen C, Tang H, Radivojac P, Arnold RJ, Zhang X, Chen JY: HIP2: an online database of human plasma proteins from healthy individuals. BMC Med Genomics 2008, 1:12.<br />
      5.	Davis AP, King BL, Mockus S, Murphy CG, Saraceni-Richards C, Rosenstein M, Wiegers T, Mattingly CJ: The Comparative Toxicogenomics Database: update 2011. Nucleic Acids Res 2011, 39(Database issue):D1067-1072.<br />
      6.	Osborne JD, Flatow J, Holko M, Lin SM, Kibbe WA, Zhu LJ, Danila MI, Feng G, Chisholm RL: Annotating the human genome with Disease Ontology. BMC Genomics 2009, 10 Suppl 1:S6.<br />
    </p>
                       
                       
                       
                       
                       
                       
                       
                       
                       
                       
                       
                       
                      </td>
                        <td width="148">
                        </td>
                    </tr>          
                </table>
                <p>&nbsp;</p>
               </div> 
          
        <!-- InstanceEndEditable -->
        
        </div>
        
        <div class="css_clear">
        </div>       
        
        <div class="css_spacing">
        </div>
        <div class="css_clear">
        </div>
        <div id="fw_footer">
            <div class="css_center">
                Copyright 2011 - <a href="http://bio.informatics.iupui.edu/index.stm">Discovery Informatics
                    and Computing Group</a>
            </div>
        </div>
    </div>
</body>
<!-- InstanceEnd --></html>
